### PR TITLE
Initial code copy

### PR DESCRIPTION
**Pull request by alexlaurinmath**
Merging should be blocked on tests passing.

### CODE REVIEW

File: fhirpath/fhirpath_test.go [hunk 1]
```diff
@@ -0,0 +1,1413 @@
...
+				Value: cpb.NameUseCode_NICKNAME,
+			},
+			Given:  []*dtpb.String{fhir.String("Senpai")},
+			Family: fhir.String("Chu"),
+		},
+		{
+			Use: &dtpb.HumanName_UseCode{
+				Value: cpb.NameUseCode_OFFICIAL,
+			},
+			Given:  []*dtpb.String{fhir.String("Kang")},
```
Comment (VickSuresh) [inline fhirpath/fhirpath_test.go:65]: Is this OK? I similarly remember there being references to Lord Voldemort and Kobe Bryant... (sorry btw lol)

**Review by bitwizeshift — COMMENTED** [review]
I really want to sarcastically ask if you can break this into smaller PRs because this is too large 🙃

**Review by bitwizeshift — APPROVED** [review]

**Review by VickSuresh — APPROVED** [review]

**Review by OliverCardoza — APPROVED** [review]
✅ Scanned list of packages
✅ Scanned for internal identifiers
✅ Scanned for copy config files / README inclusion

🚀

**Comment by alexlaurinmath** [comment]
> @alexlaurinmath is this meant to be merged into a single commit like this?
> 
> I see 7 commits in this PR, most of them are merges of the copybara branch. I thought this was preserving history?

I wasn't able to get history preservation to work. I think we see multiple commits because I'm using the same branch without deleting it? If this becomes a problem I'll either delete the branch or use a different one.

File: fhirpath/fhirpath_test.go (hunk 1 above)
Comment (alexlaurinmath) [reply fhirpath/fhirpath_test.go:65]: I think this is OK.

**Pull request merged**